src/containers/Dashboard/reducer.js

```diff
@@ -0,0 +1,32 @@
+import { fromJS } from 'immutable';
+
+import { EMPTY_ARRAY } from '../../constants/app';
+import appConstants from './constants';
+
+const initialState = fromJS({
+  cntProducts: 3,
```

Comment:
I do not think you need this field. number of products is not needed in state. You need it in the action to set how many products to fetch. And in your case it will be used in saga. 

E.g. of action call:

fetchBannerProducts({ number: 3 });

object is always more preferable, as it is more expandable in future. And we should look into future.